lib/ReactViews/Workbench/workbench-item.scss

```diff
@@ -94,4 +95,6 @@ $workbench-margin: $padding-small;
 .draggable {
   font-size: $font-size-mid-small;
   cursor: move;
+  max-width: 100%;
+  overflow-wrap: break-word;
```

Comment:
A bit nit-picky but I'd perhaps add the `overflow-wrap` to the `.nameColumn` class because that's what it's really targeting. The draggable class is more generic.

Reply:
Yup that makes sense!